data/events/2023-los-angeles.yml

```
@@ -129,10 +129,22 @@ sponsors:
     level: gold
   - id: port
     level: gold
+  - id: prosperops
+    level: gold
+  - id: mapleops
```

```suggestion
  - id: maplelabs
```

The ID used needs to match the filename for the yaml and png files that are being added.